runtime/onnxruntime/CMakeLists.txt

```diff
@@ -3,7 +3,8 @@ cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
 project(wetts VERSION 0.1)
 
 set(CMAKE_VERBOSE_MAKEFILE OFF)
-set(ONNX ON)
+option(BUILD_TESTING "whether to build unit test" OFF)
+option(ONNX "whether to build with ONNX" ON)
```

Comment:
Make http optional an compile option, since it's optional and boost is too big to  download sometimes.